README.md

```diff
@@ -5,32 +5,34 @@
 </p>
 
 [![](https://jitpack.io/v/fingerprintjs/fingerprint-android.svg)](https://jitpack.io/#fingerprintjs/fingerprint-android)
-# fingerprint-android
+# fingerprint android
 
-Simple lightweight library for device-identifying and fingerprinting, fully written in Kotlin. Crash-free. 
+Lightweight library for device identification and fingerprinting.
```

Comment:
This readme is very confusing to read, I have no understanding about subtle differences about "fingerprints", "fingerprinters", "device identification", "device fingerprinting" etc. 
I think we should develop a good glossary or a list of terms that we'll use.